src/psyclone/tests/psyir/nodes/array_mixin_test.py

```diff
@@ -491,7 +491,7 @@ def test_aref_get_full_range_unknown_size(extent):
 # _get_effective_shape
 
 
-def test_get_effective_shape(fortran_reader, fortran_writer):
+def test_get_effective_shape(fortran_reader):
```

Comment:
Good idea (and apologies for missing the previous suggestion). Added and they all work :-) The output could still be improved in a few cases but that's a nicety (and I haven't done it here because it's slightly complicated making sure it works for members of structure accesses too.)